docs/source/cloud_startup.rst

```diff
@@ -0,0 +1,90 @@
+============================================
+Cloud Startup
+============================================
+
+Overview
+********
+
+This page provides the steps necessary to get going with data ingestion and
+querying of the PDS Registry Service on the cloud from the perspective of a Discipline
+Node (DN).
+
+Initial set-up
```

Comment:
Minor typo: Better to have **set-up** start with an upper case S (as **Set-up**)to make this heading consistent with the Title Case used in other headings in this page.

Reply:
Done